src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.automationexercise.Utilities;
+
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ScreenshotsUtils
+{
+    public static  void capturescreenshot(WebDriver driver, String screenshotname) {
+
+        Path path = Paths.get("test-outputs/screenshots",screenshotname+".png");
```

Comment:
Also, here we will concatenate the TimeStamp after the screenshot name keep this in mind.

Reply:
Now or do you mean later?